akka-projection-core/src/test/scala/akka/projection/internal/metrics/ServiceTimeMetricSpec.scala

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.internal.metrics
+
+import scala.concurrent.duration._
+
+import akka.projection.HandlerRecoveryStrategy
+import akka.projection.internal.AtLeastOnce
+import akka.projection.internal.AtMostOnce
+import akka.projection.internal.ExactlyOnce
+import akka.projection.internal.FlowHandlerStrategy
+import akka.projection.internal.GroupedHandlerStrategy
+import akka.projection.internal.SingleHandlerStrategy
+import akka.projection.internal.metrics.InternalProjectionStateMetricsSpec._
+
+class ServiceTimeAndProcessingCountMetricAtLeastOnceSpec extends InternalProjectionStateMetricsSpec {
```

Comment:
This is the only suite that asserts two different metrics. Both happen on the same telemetry invocation `afterProcess` and asserting that time has passed (e.g. `instruments.lastServiceTimeInNanos.get() should be > (0L)`) is very low value anyway so I merged the two.